back-end/app.js

```diff
@@ -0,0 +1,26 @@
+var express = require('express');
+var path = require('path');
+var bodyParser = require('body-parser');
+var cookieParser = require('cookie-parser');
+var logger = require('morgan');
+
+var indexRouter = require('./routes/index');
+var usersRouter = require('./routes/users');
+
+var app = express();
```

Comment:
Here as well please. `const` or `let` instead of `var`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
This was auto-genereated by express-generator, and I did not spend any time updating the syntax in it. I can do that, though!

Reply:
Updated!